code/game/objects/items/stacks/stack_recipes.dm

```diff
@@ -92,6 +92,7 @@ var/global/list/datum/stack_recipe/metal_recipes = list (
 		new/datum/stack_recipe/chair("lime comfy chair",   /obj/structure/bed/chair/comfy/lime,   2, one_per_turf = 1, on_floor = 1),
 		new/datum/stack_recipe/chair("teal comfy chair",   /obj/structure/bed/chair/comfy/teal,   2, one_per_turf = 1, on_floor = 1),
 		new/datum/stack_recipe/chair("stool",              /obj/item/weapon/stool												   ),
+		new/datum/stack_recipe/chair("bar stool",          /obj/item/weapon/stool/bar                                              ),
```

Comment:
not really but I thought since it was pretty much a stool from above I could copy